fe/fe-core/src/main/java/org/apache/doris/nereids/stats/ExpressionEstimation.java

```diff
@@ -430,7 +457,14 @@ public ColumnStatistic visitBoundFunction(BoundFunction boundFunction, Statistic
     @Override
     public ColumnStatistic visitAggregateExpression(AggregateExpression aggregateExpression,
             Statistics context) {
-        return aggregateExpression.child().accept(this, context);
+        ColumnStatistic columnStat = aggregateExpression.child().accept(this, context);
+        if (columnStat.isUnKnown) {
+            return ColumnStatistic.UNKNOWN;
+        }
+        return new ColumnStatisticBuilder(columnStat)
+                .setMinExpr(null).setMinValue(Double.NEGATIVE_INFINITY)
```

Comment:
keep the min/max

Reply:
add AggregateFunction to disable min/max

Reply:
global agg(sum(partial_sum(A))
  +--- local agg(partial_sum(A))
if col stats of partial_sum(A) is unknown, we cannot figure out the col stats of global agg.

Reply:
will remain the original to make partial agg happy.